test/puppetlabs/i18n/core_test.clj

```diff
@@ -37,6 +47,22 @@
     (with-user-locale eo
       (is (= welcome_eo (tru welcome_en))))))
 
+(deftest test-trun
+  (testing "trun with no user locale"
+    (is (= one_bottle_en (trun one_bottle n_bottles 1)))
+    (is (= six_bottle_en (trun one_bottle n_bottles 6))))
+  (testing "trun in German"
+    (with-user-locale de (is (= one_bottle_de (trun one_bottle n_bottles 1))))
+    (with-user-locale de (is (= six_bottle_de (trun one_bottle n_bottles 6)))))
+  (testing "trun in Esperanto"
+    ;; We use Esperanto as our test locale
+    (with-user-locale eo (is (= one_bottle_eo (trun one_bottle n_bottles 1))))
+
+;; this isn't working (how should properties file be formatted?
+;;    (with-user-locale eo (is (= one_bottle_eo (trun one_bottle n_bottles 7))))
```

Comment:
Well I think it passes, because it's not actually testing against a pluralized string? It would suggest that trun if no plural form is present just defaults to the original msgid, the test string in the eo properties file.  Which makes sense as the properties file is kind of dumb.  To actually test it, one would expect to do the following.
```
(def seven_bottle_eo "fake_seven_bottle")
(with-user-locale eo (is (= seven_bottle_eo (trun one_bottle n_bottles 7))))
```
I think the question is how would one set up a properties file to actually account for msgid[1]="fake_seven_bottle" as we have in a .po file.

I also question the validity or WHAT we're testing on line 59, as it appears to just fall back to the id string and ignore all pluralization specific parameters passed in.  Don't have any answers :) Just questions.